Dockerfile

```diff
@@ -10,7 +10,11 @@ COPY apis/ apis/
 COPY controllers/ controllers/
 COPY pkg/ pkg/
 
-RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -mod vendor -o manager main.go
+RUN if [ ${ARCH}="ppc64le" ] ; then \
```

Comment:
You can simply remove `GOOS=linux GOARCH=amd64`.

Reply:
@dmage why can't we use --platform concept . 
```
FROM --platform=${BUILDPLATFORM:-linux/amd64} golang:1.17 as builder
ARG BUILDPLATFORM 
ARG TARGETARCH
RUN CGO_ENABLED=0 GOOS=linux GOARCH=${TARGETARCH} GO111MODULE=on go build -mod vendor -o manager main.go
```

I am actually planning to support quay-operator on Z & doing the same changes 

For makefile
------------
```
export DOCKER_CLI_EXPERIMENTAL=enabled

.PHONY: build # Build the container image
docker-build:
        @docker buildx create --use --name=crossplat --node=crossplat && \
        docker buildx build --platform linux/s390x,linux/amd64,linux/ppc64le  --tag $(IMG) .
```